ajax/libs/inferno-create-class/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "inferno-create-class",
+  "filename": "inferno-create-class.min.js",
+  "version": "3.7.0",
+  "license": "MIT",
+  "description": "Provides a helper to create Inferno Components without needing ES2015",
+  "author": {
+    "name": "Dominic Gannaway",
+    "email": "dg@domgan.com"
+  },
+  "homepage": "https://infernojs.org/",
+  "keywords": [
+    "babel",
+    "react",
+    "inferno",
+    "framework",
+    "interfaces",
+    "user interfaces"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/infernojs/inferno.git"
+  },
+  "npmName": "inferno-create-class",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/inferno-create-class!(*.node)*.+(js|css|map)"
```

Comment:
You should exclude the file type we won't need here, not the include js, css, map only, we should assume most of the files should be useful. @sufuf3 @extend1994 I think you both know that?

Reply:
Yes, it should be like that. Because the pattern `+(js|css|map)` was used and merged in the last few inferno related library, I thought the pattern is okay. Now I am going to modify it, which makes more sense. 
Should I also modify the pattern of inferno-router, inferno-hyperscript, inferno-devtools, inferno-server, inferno-shared, inferno-create-element, inferno-test-utils, inferno-component, inferno-mobx, inferno-redux? All of them use this kind of pattern.

Reply:
As they are currently working, to save time, sometimes I won't request to change immediately to all the cases, but it's good to use the new pattern, you can use a single commit to update them all! (No need to use multi commits here)